Command failed: python "d:/ASZHANMIKE/OneDrive - Carlsberg Group/MyODDocs/AlwaysCode/CodeforcesPython/Codeforces Round 810 Div. 2/B_Party/B_Party.py" 1  < "d:/ASZHANMIKE/OneDrive - Carlsberg Group/MyODDocs/AlwaysCode/CodeforcesPython/Codeforces Round 810 Div. 2/B_Party/Tests/input_1.txt" > "d:/ASZHANMIKE/OneDrive - Carlsberg Group/MyODDocs/AlwaysCode/CodeforcesPython/Codeforces Round 810 Div. 2/B_Party/Tests/code_output_1.txt"
Traceback (most recent call last):
  File "d:\ASZHANMIKE\OneDrive - Carlsberg Group\MyODDocs\AlwaysCode\CodeforcesPython\Codeforces Round 810 Div. 2\B_Party\B_Party.py", line 6, in <module>
    b = [0] * n1
NameError: name 'n1' is not defined. Did you mean: 'n'?
